src/bp/core/api.ts

```diff
@@ -221,6 +222,14 @@ const workspaces = (workspaceService: WorkspaceService): typeof sdk.workspaces =
   }
 }
 
+const distributed = (jobService: JobService): typeof sdk.distributed => {
```

Comment:
what do you think if we used  `Job` as namespace instead of distributed. It would follow the current naming "convention"